org.vergecurrency.verge-qt.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally should be shipped in your tarball/archive for others to consume.

Reply:
will look into that, certainly seems feasible.